catalog/pipeline/fbc/0.1/fbc.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: bundle-fbc-pipelinerun
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: fbc
+spec:
+  description: >-
+    Tekton pipeline to interact with IIB service for File Based Catalogs
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format
+    - name: fbcServiceUrl
```

Comment:
This should be IIBServiceUrl. FBC is not a service

Reply:
Ok, all places referring FBC as service were changed to IIB. Please take a look again @scoheb